.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,8 +6,36 @@ about: Create a report to help reproduce and fix the issue
 
 <!-- If you're certain the issue is to be caused by a test merge, report it in the pull request's comment section rather than on the tracker(If you're unsure you can refer to the issue number by prefixing said number with #. The issue number can be found beside the title after submitting it to the tracker).If no testmerges are active, feel free to remove this section. -->
 
-## Reproduction:
+## Description
+<!-- a quick, 1-2 sentence summary -->
 
-<!-- Explain your issue in detail, including the steps to reproduce it. Issues without proper reproduction steps or explanation are open to being ignored/closed by maintainers.-->
 
-<!-- **For Admins:** Oddities induced by var-edits and other admin tools are not necessarily bugs. Verify that your issues occur under regular circumstances before reporting them. -->
+## Difference between expected and actual behavior
+<!-- what does it do, what it should do... -->
+
+
+## Steps to reproduce
+<!-- step by step on how we can reproduce this issue, as detailed as possible -->
+
+
+## Specific information for locating
+<!-- e.g. an object name, paste specific message outputs... -->
+
+
+## Length of time in which bug has been known to occur
+<!--
+	Be specific if you approximately know the time it's been occurring
+	for—this can speed up finding the source. If you're not sure
+	about it, tell us too!
+-->
+
+
+## Issue bingo
+<!-- Check these by writing an x inside the [ ] (like this: [x])-->
+<!-- Don't forget to remove the space between the brackets, or it won't work! -->
+- [ ] Issue could be reproduced at least once
+- [ ] Issue could be reproduced by different players
+- [ ] Issue could be reproduced in multiple rounds
+- [ ] Issue happened in a recent (less than 7 days ago) round
+- [ ] Couldn't find an existing issue about this (https://github.com/cmss13-devs/cmss13/issues)
```

Comment:
People should report bugs even if they can't tick every checkbox, this might discourage them from doing so if they have to do extra work to get their bug reported

Reply:
might be worth removing these checkboxes:
- [ ] Issue could be reproduced in multiple rounds
- [ ] Issue could be reproduced by different players